onnx/utils.py

```diff
@@ -21,3 +24,138 @@ def polish_model(model):  # type: (ModelProto) -> ModelProto
     model = onnx.optimizer.optimize(model)
     onnx.checker.check_model(model)
     return model
+
+
+class Extractor:
+    def __init__(self, model):  # type: ignore
+        self.model = onnx.shape_inference.infer_shapes(model)
+        self.graph = self.model.graph
+        self.wmap = self._buildNameDict(self.graph.initializer)
+        self.vimap = self._buildNameDict(self.graph.value_info)
+
+    @staticmethod
+    def _buildNameDict(objs):  # type: ignore
```

Comment:
adapt similar naming convention as other python files, basically this should be _build_name_dict